.editorconfig

```diff
@@ -1,8 +1,12 @@
 root=true
 
 [*]
-indent_style = space
-indent_size = 4
 charset = utf-8
+max_line_length = 120
```

Comment:
The default line-length for prettier is 80, and it isn't being overridden in the .prettierrc, so I guess it should be 80 here as well, right?

Reply:
@IanVS no, we are using the Prettier from ESLint:

https://github.com/eirslett/storybook-builder-vite/blob/34a690bcb6e2ac1f7a8810b47e748574ca5434ca/.eslintrc.js#L12-L13

Reply:
Updated `.prettierrc` in 12279cd.

Reply:
Actually it's not necessary to reflect this setting; prettier reads `.editorconfig` files and respects them. Pretty convenient. https://github.com/prettier/prettier/blob/514046b3c70e6477cb69b4d36871d0d4c8c5e415/src/config/resolve-config-editorconfig.js

Reply:
@eirslett Okay, I can remove setting from `.prettierrc`.

But I will keep it in `.eslintrc`, because we need to multiple vertical lines in IDEs.

Reply:
> But I will keep it in .eslintrc, because we need to multiple vertical lines in IDEs.

Or maybe use something like this - I've never tried it myself though. https://www.npmjs.com/package/eslint-plugin-editorconfig

Reply:
Fixed in 4ce1418.